### ISSUE TITLE
For >=5.0.0, pypi wheel incorrectly states that py2 is supported

### ISSUE THREAD

**Issue by erickim555** [grantjenks/python-diskcache#155]
In the pypi wheel for diskcache >=5.0.0, the wheel claims that both py2 and py3 are supported, when in fact py2 support was dropped in 5.0.0: 

https://pypi.org/project/diskcache/#files

Can you please update the pypi wheel to indicate that py2 is not supported for `diskcache>=5.0.0`? 

For what it's worth, we are seeing this syntax error after our py2 build systems upgraded diskcache from 4.0.0 to 5.0.0:
```
>>> import diskcache
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/diskcache/__init__.py", line 9, in <module>
    from .core import (
  File "/usr/local/lib/python2.7/dist-packages/diskcache/core.py", line 434
    raise ValueError('disk must subclass diskcache.Disk') from None
                                                             ^
SyntaxError: invalid syntax
```

Thank you!

**Comment by grantjenks** [comment]
Oops, sorry about that. I will try to fix today

**Comment by grantjenks** [comment]
Fixed in version 5.0.3, just released on PyPI

**Comment by impredicative** [comment]
> Fixed in version 5.0.3, just released on PyPI

That's incorrect. You mean 5.0.2.
